public/app/features/plugins/built_in_plugins.ts

```diff
@@ -13,8 +13,7 @@ const grafanaPlugin = async () =>
 const influxdbPlugin = async () =>
   await import(/* webpackChunkName: "influxdbPlugin" */ 'app/plugins/datasource/influxdb/module');
 const lokiPlugin = async () => await import(/* webpackChunkName: "lokiPlugin" */ 'app/plugins/datasource/loki/module');
-const jaegerPlugin = async () =>
-  await import(/* webpackChunkName: "jaegerPlugin" */ 'app/plugins/datasource/jaeger/module');
+const jaegerPlugin = async () => await import(/* webpackChunkName: "jaegerPlugin" */ '@grafana-plugins/jaeger/module');
```

Comment:
Please remove this - since https://github.com/grafana/grafana/pull/81873 these shouldn't exist as plugins are always loaded from the built `dist/module.js` file.